packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -150,13 +153,19 @@ contract BridgePool is MultiCaller, Testable, BridgePoolInterface, ERC20, Lockab
     event RelayCanceled(bytes32 indexed depositHash, bytes32 indexed relayHash, address indexed disputer);
     event RelaySettled(bytes32 indexed depositHash, address indexed caller, RelayData relay);
     event BridgePoolAdminTransferred(address oldAdmin, address newAdmin);
+    event RelaysEnabledSet(bool newRelaysEnabled);
     event LpFeeRateSet(uint64 newLpFeeRatePerSecond);
 
     modifier onlyBridgeAdmin() {
         require(msg.sender == address(bridgeAdmin), "Caller not bridge admin");
         _;
     }
 
+    modifier onlyIfRelaysEnable() {
```

Comment:
nit: rename to `onlyIfRelaysEnabled`. Missing "d" :D

Reply:
fixed